neptune/handler.py

```diff
@@ -0,0 +1,79 @@
+#
+# Copyright (c) 2020, Neptune Labs Sp. z o.o.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from typing import List, TYPE_CHECKING
+
+from neptune.internal.utils import parse_path, path_to_str
+from neptune.variable import FloatVariable, StringVariable, FloatSeriesVariable, StringSeriesVariable, StringSetVariable
+
+if TYPE_CHECKING:
+    from neptune.experiment import Experiment
+
+
+class Handler:
+
+    def __init__(self, _experiment: 'Experiment', path: List[str]):
+        super().__init__()
+        self._experiment = _experiment
+        self._path = path
+
+    def __getitem__(self, path: str) -> 'Handler':
+        return Handler(self._experiment, self._path + parse_path(path))
+
+    def __setitem__(self, key: str, value) -> None:
+        self[key].assign(value)
+
+    def __getattr__(self, attr):
```

Comment:
So we also support attribute syntax, not just dictionary lookup syntax?

How about paths which are not valid Python attribute names?

Reply:
It is not used like that. It proxies method call to underlying variable. So if FloatVariable has a `get` mathod we can call `exp['var'].get()` on a `Handler` object.